packages/frontend/web-ui/src/scss/home/pages/home/_page.scss

```diff
@@ -45,3 +45,7 @@
 .login-page-container {
   min-height: 100vh;
 }
+
+.forgot-page-container {
```

Comment:
This is not the right place

Reply:
I've moved login-page-container also to their file scss.